libs/build_netcdf.sh

```diff
@@ -150,10 +152,10 @@ set -x
 
 if [[ ${STACK_netcdf_shared} =~ [yYtT] ]]; then
   export LIBS=$($prefix/bin/nc-config --libs)
-  export LDFLAGS+=" -L$prefix/lib"
+  export LDFLAGS+=" -L$prefix/lib -lnetcdf -lhdf5_hl -lhdf5 -lz"
```

Comment:
These should not be required because `nc-config --libs` should be giving everything except the path to the newly build netcdf-c library

Reply:
See above comments.